inc/sairouterinterface.h

```diff
@@ -239,6 +239,19 @@ typedef enum _sai_router_interface_attr_t
      */
     SAI_ROUTER_INTERFACE_ATTR_LOOPBACK_PACKET_ACTION,
 
+    /**
```

Comment:
++ specific comments to not allow nbr entries , and this is simply about resource optimization. 
++ comment for using the native rif can also be used for vrrp , simply not using the egress resouces or not learning nbr on this vrrp rif.

Reply:
++ on virtual rif created by using this flag, attribute are invalid - ADMIN state for v4 and v6, mtu size, v4 and v6 mcast enable, neighbor packet actions. 

@lguohan and @AshokDaparthi  please check.